<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scheduler</class>
 <widget class="QMainWindow" name="Scheduler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The Academic Scheduler</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="duration_lecture">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>42</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="date_lecture">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy/M/d</string>
    </property>
    <property name="date">
     <date>
      <year>2024</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QTimeEdit" name="time_lecture">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>118</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lecture_description">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_lecture_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Info</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_schedule">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>421</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lecture Name : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Time :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Date :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 20pt &quot;Vampire Wars&quot;;</string>
    </property>
    <property name="text">
     <string>Welcome To The Academic Scheduler</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>331</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid black;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>Subject :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="deadline_description">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0.5px solid black;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>Deadline :</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="date_deadline">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0.5 solid black;</string>
     </property>
     <property name="displayFormat">
      <string>yyyy/M/d</string>
     </property>
     <property name="date">
      <date>
       <year>2024</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
    <widget class="QPushButton" name="add_deadline_button">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0.5px solid black;</string>
     </property>
     <property name="text">
      <string>Add Deadline</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>533</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>85</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Vampire Wars&quot;;</string>
    </property>
    <property name="text">
     <string>To Set Deadlines :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>49</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hrs</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
